docs/docs/external-volumes.md

```diff
@@ -0,0 +1,137 @@
+---
+title: Stateful Applications Using External Persistent Volumes
+---
+
+# Stateful Applications Using External Persistent Volumes
+
+<div class="alert alert-danger" role="alert">
+  <span class="glyphicon glyphicon-exclamation-sign" aria-hidden="true"></span> Adapted in Marathon Version 1.0 <br/>
+  The Persistent Storage functionality is considered beta, so use this feature at your own risk. We might add, change, or delete any functionality described in this document.
+</div>
+
+Marathon applications normally lose their state when they terminate and are relaunched. In some contexts, for instance, if your application uses MySQL, you’ll want your application to preserve its state. You can use an external storage service, such as Amazon's Elastic Block Store (EBS), to create a persistent volume that follows your application instance.
+
+Using an external storage service allows your apps to be more fault-tolerant. If a host fails, Marathon reschedules your app on another host, along with its associated data, without user intervention.
+
+# Specifying an External Volume
+
+If you are running Marathon on DCOS, add the following to the `genconf/config.yml` file you use during DCOS installation. [Learn more](https://docs.mesosphere.com/concepts/installing/installing-enterprise-edition/configuration-parameters/).
```

Comment:
Can you configure an existing DCOS cluster to use rexray, or do you have to start a fresh cluster?
How can you configure this without DCOS? Remember, Marathon is not just used for DCOS.
Should open-source Marathon be linking to DCOS EE docs?